meson_options.txt

```diff
@@ -34,6 +35,7 @@ option('use_sys_libuv', type: 'feature', value: 'auto', description: 'Whether to
 option('use_libuv', type: 'boolean', value: true, description: 'If true, libuv is used to handle remote features')
 option('use_swift_demangler', type: 'boolean', value: true, description: 'If false, disables the swift demangler')
 option('use_gpl', type: 'boolean', value: true, description: 'Set to false when you want to disable gpl code')
+option('install_sigdb', type: 'boolean', value: false, description: 'Downloads and installs rizin sigdb')
```

Comment:
shall we have this as `feature`? We can have `auto`, `true`,`false`. Maybe `auto` can default to `true` in case it is a release buildtype, false otherwise? cc @wargio @thestr4ng3r @Maijin @XVilka

Reply:
i think false is enough, no need to have that, because i believe this has to be explicitly defined.

Reply:
depends on how many users are using git version or release these days tbh